libcore/marlin-file-operations.c

```diff
@@ -802,6 +807,18 @@ custom_basename_to_string (char *format, va_list va)
     return name;
 }
 
+static char *
+custom_basename_to_string (char *format, va_list va)
+{
+    GFile *file;
+    GFileInfo *info;
+    char *name, *basename, *tmp;
```

Comment:
I suppose `info`, `name`, `basename`, and `tmp` are not needed in this function?